README.md

```diff
@@ -59,7 +59,9 @@ You’ll be asked to install the [DatoCMS](https://www.datocms.com) integration.
 
 ### CMS
 
-Feel free to modify the code to use a different content management system. See `lib/cms-api.ts` for details.
+Environment variables determine which CMS to use. See `lib/cms-api.ts` for details and `.env.local.example` for all environment variables. The demo ([demo.vercel.events](https://demo.vercel.events)) uses DatoCMS, but we also have support for:
+
+- [Contentful](https://github.com/vercel/virtual-event-starter-kit/tree/main/lib/cms-providers/contentful.ts)
```

Comment:
Relative links work on GitHub: https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-readmes#relative-links-and-image-paths-in-readme-files

```suggestion
- [Contentful](lib/cms-providers/contentful.ts)
```